nucypher/policy/conditions/auth/evm.py

```diff
@@ -85,12 +85,14 @@ def authenticate(cls, data, signature, expected_address):
             )
 
         # enforce a freshness check
-        # TODO: "not-before" throws off the freshness timing; so skip if specified. Is this safe / what we want?
+        # TODO: "not-before" throws off the freshness timing; so skip if specified.
+        #  Is this safe / what we want?
         if not siwe_message.not_before:
             issued_at = maya.MayaDT.from_iso8601(siwe_message.issued_at)
             if maya.now() > issued_at.add(hours=cls.FRESHNESS_IN_HOURS):
                 raise cls.AuthenticationFailed(
-                    f"EIP4361 message is stale; more than {cls.FRESHNESS_IN_HOURS} hours old (issued at {issued_at.iso8601()})"
+                    f"EIP4361 message is stale; more than {cls.FRESHNESS_IN_HOURS} "
+                    f"hours old (issued at {issued_at.iso8601()})"
```

Comment:
Just a minor suggestion but "expired" comes off a bit more naturally than the shop-talk of "stale".

Reply:
I hesitated to use "expired" because this is our own expiry check. EIP4361 has its own concept of `expiration-time` - https://docs.login.xyz/general-information/siwe-overview/eip-4361#message-field-descriptions.

In addition, we are implementing our own 2-hour freshness check, which still raises an error even if the SIWE message's expiry date is further in the future.

I'm still uncertain about this two-hour check. Its motivation is rooted in preventing an SIWE message from being indefinitely reused by not having an expiration time or an expiration time that is really far in the future.